client/src/infographic/Infographic.js

```diff
@@ -227,15 +261,60 @@ class InfoGraph_ extends React.Component {
               }
             </p>
           }
-          { !loading &&
-            _.map(panel_keys, panel_key => 
-              <PanelRenderer 
-                panel_key={panel_key}
-                subject={subject}
-                active_bubble_id={active_bubble_id}
-                key={panel_key + subject.guid}
-              />
-            )  
+          { is_filter_panels &&
+            <Details
+              closed_drawer_icon={
+                <IconFilter
+                  {...filter_icon_props}
+                  key="closed_filter"
+                />
+              }
+              opened_drawer_icon={
+                <IconFilter
+                  {...filter_icon_props}
+                  key="opened_filter"
+                  rotation={180}
+                />
+              }
+              summary_content={
+                <div>
+                  <span style={{fontSize: 16}}>
+                    <TM k="filter_panels"/>
+                  </span>
+                  <span> </span>
```

Comment:
Not sure how I should separate this two text. Is this good enough?

Reply:
Kind of unrelated to your exact question but I just added something, in a PR merged to master today, that helps here a bit. If you rebase, you'll be able to pass `style` and `className` props directly to `TM`s., so you won't need to wrap them in bonus `span`s.

I think you could also get the space in as a text node by replacing `<span> </span>` with `{" "}`. Still a bit wonky, but at least clearer that there's an actual space character being inserted. Can't think of anything better than that right now.